<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TablePageTurnSwitch</class>
 <widget class="QWidget" name="TablePageTurnSwitch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1696</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 116px;
height: 19px;
font-family: PingFang SC;
font-weight: bold;
font-size: 20px;
color: #666666;
line-height: 121px;</string>
       </property>
       <property name="text">
        <string>共</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_all">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 116px;
height: 19px;
font-family: PingFang SC;
font-weight: bold;
font-size: 20px;
color: #37acba;
line-height: 121px;</string>
       </property>
       <property name="text">
        <string>10000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 116px;
height: 19px;
font-family: PingFang SC;
font-weight: bold;
font-size: 20px;
color: #666666;
line-height: 121px;</string>
       </property>
       <property name="text">
        <string>条记录</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>714</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_left">
       <property name="minimumSize">
        <size>
         <width>58</width>
         <height>58</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>58</width>
         <height>58</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 58px;
height: 58px;
background: #22A4B3;
border-radius: 29px;
font-size: 12px;
font-weight: bold;
color: #000000;

image: url(:/suzhou/btn_arrow_left.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_msg">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="toolTipDuration">
        <number>-4</number>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 402px;
height: 58px;
background: #F2F7FB;
border-radius: 29px;
border: 1px solid rgba(170,189,212,0.6);
font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_right">
       <property name="minimumSize">
        <size>
         <width>58</width>
         <height>58</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>58</width>
         <height>58</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 58px;
height: 58px;
background: #22A4B3;
border-radius: 29px;
font-size: 12px;
font-weight: bold;
color: #000000;

image: url(:/suzhou/btn_arrow_right.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
